backend/coins/btc/handlers/handlers.go

```diff
@@ -55,7 +55,8 @@ type Handlers struct {
 
 // NewHandlers creates a new Handlers instance.
 func NewHandlers(
-	handleFunc func(string, func(*http.Request) (interface{}, error)) *mux.Route, log *logrus.Entry) *Handlers {
```

Comment:
This (and another change below) are done by `gofmt`, I might have a version different from the rest of the team, I'll check that soon